contracts/introspection/SupportsInterfaceWithLookup.sol

```diff
@@ -10,7 +10,7 @@ import "./IERC165.sol";
  */
 contract SupportsInterfaceWithLookup is IERC165 {
 
-  bytes4 public constant InterfaceId_ERC165 = 0x01ffc9a7;
+  bytes4 private constant InterfaceId_ERC165 = 0x01ffc9a7;
```

Comment:
is there a reason for this one to be public?

Reply:
Not `public`, but we have been making `InterfaceId_*` variables `internal` so that inheriting contracts can register that they support those interfaces.

Reply:
but in this case, the constructor is registering that interface. What is left for inheriting contracts to do?

Reply:
Ah, right. The thing is `InterfaceId_*` variables are generally declared in the interface, so they need to be `internal` for the constructor of the implementation to use them. See #1065 for example. But I'm doubting if this is the right thing. I'd say leave this `private` and I'll make a note to check on this later.